setup.cfg

```diff
@@ -11,44 +11,23 @@ classifier =
     Topic :: Internet :: WWW/HTTP :: Dynamic Content
     Intended Audience :: Developers
     Programming Language :: Python
-    Programming Language :: Python :: 2
-    Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3.5
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
     Development Status :: 5 - Production/Stable
     Operating System :: OS Independent
 
-[coverage:run]
-branch = True
-source = stor
-omit = stor/tests/test_posix_path_compat.py,stor/tests/test_integration_swift.py,stor/tests/test_integration_s3.py,stor/tests/test_integration.py,stor/third_party/backoff.py,stor/tests/test_integration_dx.py
-
-[coverage:report]
-exclude_lines =
-    # Have to re-enable the standard pragma
-    pragma: no cover
-    raise NotImplementedError
-fail_under = 100
-show_missing = 1
-
 [files]
 packages = stor
-scripts = stor/stor-completion.bash
-
-[entry_points]
-console_scripts =
-    stor = stor.cli:main
 
+# can't move because flake8 does not yet support pyproject.toml
 [flake8]
-max-complexity=10
+max-complexity=13
 max-line-length=99
 exclude = stor/third_party/*
-
-[pbr]
-skip_authors = true
-skip_changelog = true
+ignore=E226,W504
```

Comment:
were these causing many issues? I personally like it

Reply:
Like I said, linter had been swallowed for a while. You are totally welcome if you want to remove this and fix lint issues that arise :)